posts/test_runner.py

```diff
@@ -0,0 +1,33 @@
+"""Unit test runner for users service.
```

Comment:
should be posts service, not users

Reply:
Thanks, fixed in https://github.com/knative-portability/large-events/pull/10/commits/a7636bc15b2d823226ae2e327b89bee1b3237fbf